tests/cases/fourslash/server/jsdocCallbackTag.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference path="../fourslash.ts"/>
+
+// @allowNonTsExtensions: true
+// @Filename: jsdocCallbackTag.js
+
+//// /**
+////  * @callback FooHandler
+////  * @param {string} eventName

```

Comment:
can you add some documentation? I don't think it'll work now, but once my jsdoc change is also merged, then `parseParamTag` should also parse documentation. In the meantime, it should be skipped correctly.